plugin/src/main/kotlin/nl/avisi/anowalke/toolWindow/FluffyListener.kt

```diff
@@ -48,66 +48,64 @@ class FluffyListener(val project: Project) : ToolWindowManagerListener {
     }
 
     fun scan(result: JTextArea) {
-        log.warn("Initiate scan function.")
+        log.info("Initiate scan function.")
         val identifiers = scanProject()
         val stemmedIdentifiers: List<Identifier> = identifiers.map {
             Identifier(it.className, stemmer.stem(it.expression))
         }
-        stemmedIdentifiers.forEach { log.warn("Stemmed identifier: $it") }
+        stemmedIdentifiers.forEach { log.info("Stemmed identifier: $it") }
         val results = findShotgunSurgery(stemmedIdentifiers)
         result.text = results.toString()
     }
 
     private fun getJavaFiles(file: VirtualFile): List<Identifier>? {
-        log.warn("Initiate getJavaFiles function.")
+        log.info("Initiate getJavaFiles function.")
         // Filter out hidden files
         if (!file.isDirectory && !file.name.startsWith(".")) {
-            log.warn("File: ${file.name} is not a directory nor a hidden file.")
-            var expressions = PsiManager.getInstance(project).findFile(file)?.let { psiFile ->
+            log.info("File: ${file.name} is not a directory nor a hidden file.")
+            val expressions = PsiManager.getInstance(project).findFile(file)?.let { psiFile ->
                 psiService.extract(psiFile)
             }
             if (expressions != null) {
-                log.warn("Extract function retrieved ${expressions.size} expressions.")
-                expressions = sanitiser.sanitise(expressions)
+                log.info("Extract function retrieved ${expressions.size} expressions.")
+                return sanitiser
```

Comment:
Nog mooier is dan om de return ook buiten de if te zetten. Ik gok dat IntelliJ daar ook over begint

Reply:
Ja deze snap ik niet precies hoe je dat hier bedoelt..